server/src/db/seeders/UserSeeder.js

```diff
@@ -0,0 +1,32 @@
+import { User } from "../../models/index.js"
+
+class UserSeeder {
+    static async seed() {
+        const usersData = [
+          {
+            email: "tess.ting@gmail.com",
+            userName: "testtess",
+            cryptedPassword: "haioejfaheignbkn238",
```

Comment:
if you instead set up the user seeder with `password` property, you will be able to log in as these fake users. otherwise you can't sign in as these accounts!